main.c

```diff
@@ -460,45 +466,57 @@ static void parse_args(int argc, char *const argv[], struct options *options)
         case 'h':
             usage(argv[0]);
             exit(EXIT_SUCCESS);
+            break;
         case 'v':
             version();
             exit(EXIT_SUCCESS);
+            break;
         default:
-            usage(argv[0]);
-            exit(EXIT_FAILURE);
+            goto error;
+            break;
```

Comment:
the break here is not necessary.

What was wrong with the original code?  Does clang complain about it?

Reply:
the break is not necessary here but personally I prefer to have `break` everywhere.

The clang-tidy error is shown in #112 and fixed by moving `strdup(optarg)` out of the `while(getopt())` loop.